tk_toolchain/cmd_line_tools/tk_docs_generation/sphinx_processor.py

```diff
@@ -126,17 +126,15 @@ def build_docs(self, name, version, warnings_as_errors=True):
         # run build command
         # Use double quotes to make sure it works on Windows and Unix.
         # -T means show traceback
-        cmd = (
-            'sphinx-build -c "%s" %s -T -E -D project="%s" -D release="%s" -D version="%s" "%s" "%s"'
-            % (
-                self._sphinx_conf_py_location,
-                warnings_as_errors_flag,
-                name,
-                version,
-                version,
-                self._docs_path,
-                self._sphinx_build_dir,
-            )
+        cmd = '%s -m sphinx -c "%s" %s -T -E -D project="%s" -D release="%s" -D version="%s" "%s" "%s"' % (
```

Comment:
Python scripts should always be invoked using `python -m` in case the script is not in the path.

Reply:
This was actually broken on my computer because sphinx-build was not in `/usr/local/bin`